app/services/notificationService.ts

```diff
@@ -140,8 +140,8 @@ export const getCDGWNotisForChild = (childTaxonomyData: any, child: any, prematu
   // noti.push(//at the begginning of the period
   //if> 3months then add noti
   //check gap between days_to and days_from
-  const childBirthDatePlanned = child?.taxonomyData.prematureTaxonomyId != null && child?.taxonomyData.prematureTaxonomyId != "" && child?.taxonomyData.prematureTaxonomyId != undefined ? child.plannedTermDate : child.birthDate;
-  const activityTaxonomyId = child?.taxonomyData.prematureTaxonomyId != null && child?.taxonomyData.prematureTaxonomyId != undefined && child?.taxonomyData.prematureTaxonomyId != "" ? child?.taxonomyData.prematureTaxonomyId : child?.taxonomyData.id;
+  const childBirthDatePlanned = child?.taxonomyData?.prematureTaxonomyId ?? child?.birthDate;

```

Comment:
Updated code is not the same as it was before, please make sure it is intentional.

Reply:
yes it's intentional it's optimise condition using nullish coalescing operator